src/onegov/org/cronjobs.py

```diff
@@ -826,29 +827,34 @@ def delete_unconfirmed_newsletter_subscriptions(request: OrgRequest) -> None:
 
 
 def get_news_for_push_notification(session: Session) -> Query[News]:
+    # Use UTC time for database comparisons since publication_start is stored
+    # in UTC
     now = utcnow()
-    ten_minutes_ago = now - timedelta(minutes=10)
 
     # Get all news items that should trigger push notifications
     query = session.query(News)
-    query = query.filter(News.published == True)
+    query = query.filter(News.published.is_(True))
     query = query.filter(News.publication_start <= now)
+    query = query.filter(News.publication_start.isnot(None))
+
+    news_with_sent_notifications = session.query(
+        PushNotification.news_id
+    ).subquery()
+    query = query.filter(~News.id.in_(news_with_sent_notifications))
+    only_public_news = query.filter(
+        or_(
+            News.meta['access'].astext == 'public',
+            News.meta['access'].astext == None,
+        )
+    )
 
-    # You may comment out the line below temporarily for testing
-    query = query.filter(News.publication_start >= ten_minutes_ago)
-
-    only_public_news = query.filter(or_(
-        News.meta['access'].astext == 'public',
-        News.meta['access'].astext == None
-    ))
     only_public_with_send_push_notification = only_public_news.filter(
         News.meta['send_push_notifications_to_app'].astext == 'true'
     )
-
     return only_public_with_send_push_notification
 
 
-@OrgApp.cronjob(hour='*', minute='*/10', timezone='UTC')
+@OrgApp.cronjob(hour='*', minute='*/10', timezone='Europe/Zurich')
```

Comment:
I would stick to `UTC` here, since it runs every 10 minutes anyways. Less likely to cause issues on days where we switch to or from daylight savings time.